pkg/lib/cockpit-components-dialog.jsx

```diff
@@ -235,8 +235,7 @@ class Dialog extends React.Component {
             </Popover>;
 
         const error = this.props.error || this.props.static_error;
-        const error_alert = error &&
-            <Alert variant='danger' isInline title={React.isValidElement(error) ? error : error.toString() }>{error.details}</Alert>;
+        const error_alert = error && <Alert variant='danger' isInline title={error} />;
```

Comment:
Looking at the [PF docs](https://www.patternfly.org/components/alert#alert), this can only take an React element

Reply:
Sure, but a string is a valid React node. The examples on the page are plain strings.

Reply:
Ahh, TIL!